docs/DeveloperGuide.md

```diff
@@ -403,10 +449,15 @@ Use case ends.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
 5. Should display response within 1 second.
-6. Should work stand-alone.
+6. Should work stand-alone and should not be a plug-in to another software.
 7. Should work without internet connection.
 8. Should store data in text file.
 9. Should work without requiring an installer.
+10. Should work only with Command Line Interface as the main input with minimal use of mouse/clicking.
```

Comment:
Isn't this the reason why the project is the way it is ? The motivation of the application is difficult to defend as a NFR.